src/js/helpers/graph/declarationHelper.js

```diff
@@ -32,14 +32,27 @@ export const getNodeGroup = (node) => {
   return node.status.toLowerCase();
 };
 
-const createVisNode = (node) => ({
-  id: node.id,
-  label: node.label,
-  group: getNodeGroup(node),
-  info: {
-    ...node.info,
-  },
-});
+const createVisNode = (node) => {
+  const info = node.status.toLowerCase() === 'missing'
```

Comment:
Missing node is always SINGLE.

I think we need to handle all SINGLE nodes (not only missing ones).

Reply:
for missing node we set custom log. For single we delete subtasks. So we need two diffrent functionality. we can only rewrite this to 

```
if single node {
    if missing node
       do something
    else
        do something
}
```

But i don't know this notation is better. Result is the same.

Reply:
ok